docs/book/src/developer/tilt.md

```diff
@@ -262,12 +262,14 @@ With this config, the respective managers will be invoked with:
 manager --logging-format=json
 ```
 
-### Run Tilt!
+### Create a kind cluster and run Tilt!
 
-To launch your development environment, run
+### Create a kind cluster
```

Comment:
Should we drop this headline given we already have 265? (and they are both on the same level)

Reply:
+1 to dropping.